roles/organizations/README.md

```diff
@@ -1,23 +1,20 @@
 # foreman.organizations
+
 ## Description
 An Ansible Role to create Organizations in Foreman.
 
-## Requirements
-
 ## Variables
 |Variable Name|Default Value|Required|Description|Example|
 |:---:|:---:|:---:|:---:|:---:|
-|`tower_state`|"present"|no|The state all objects will take unless overridden by object default|'absent'|
-|`tower_hostname`|""|yes|URL to the Ansible Tower Server.|127.0.0.1|
-|`validate_certs`|`False`|no|Whether or not to validate the Ansible Tower Server's SSL certificate.||
-|`tower_username`|""|yes|Admin User on the Ansible Tower Server.||
-|`tower_password`|""|yes|Tower Admin User's password on the Ansible Tower Server.  This should be stored in an Ansible Vault at vars/tower-secrets.yml or elsewhere and called from a parent playbook.||
-|`tower_oauthtoken`|""|yes|Tower Admin User's token on the Ansible Tower Server.  This should be stored in an Ansible Vault at or elsewhere and called from a parent playbook.||
-|`tower_organizations`|`see below`|yes|Data structure describing your organzation or organizations Described below.||
+|`foreman_server_url`|"present"|no|URL of the Foreman server
+|`foreman_username`|""|yes|Username accessing the Foreman Server.||
```

Comment:
username and password should not be required. check code for `default(omit)` to verify